### PR TITLE
Set new publishing paradigm

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: db/migrate/20210511212929_change_to_single_iteration_publiushing.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class ChangeToSingleIterationPubliushing < ActiveRecord::Migration[6.1]
```
Comment (ErikSchierboom) [inline db/migrate/20210511212929_change_to_single_iteration_publiushing.rb:1]: You'll probably squash all the migrations anyway, but 🤷 

```suggestion
class ChangeToSingleIterationPublishing < ActiveRecord::Migration[6.1]
```

**Review by ErikSchierboom — APPROVED** [review]

**Comment by kntsoriano** [comment]
@iHiD Should be ready :)

**Comment by iHiD** [comment]
Excellent. I shall get reviewing! (I've rebased and force-pushed FYI)

**Comment by iHiD** [comment]
@kntsoriano I've got this looking good now. All your code worked great :) I've made one change which is to remove the second form with the buttons as I felt it made more sense to have the same experience in both. Might be worth just double checking my commits to make sure you're happy with it all :) Great work.

**Pull request merged**